src/main/scala/sigmastate/serialization/DataJsonEncoder.scala

```diff
@@ -36,6 +36,15 @@ object DataJsonEncoder {
     ErgoAlgos.encode(bytes).asJson
   }
 
+  def encode(v: AnyValue): Json = {
```

Comment:
to be consistent with the method below it should be named `encodeAnyValue`

Reply:
Renamed